### DISCUSSION TITLE
Check SHA's before pulling locally

### DISCUSSION THREAD

**Discussion by garfbradazEQ** [microsoft/dotnet-framework-docker#1024] [Q&A] [answered]
Where can I check the SHA _before_ pulling down locally? I cannot see them on the docker hub like I can for other images.

![image](https://user-images.githubusercontent.com/117753795/201078822-249bc41c-d5be-4778-aaf3-313b2ed43846.png)

For example ubuntus is shown:

![image](https://user-images.githubusercontent.com/117753795/201078909-d650443c-ec6a-4215-9eb3-6efd2814fbdc.png)

**Comment by mthalman** [comment, marked as answer]
You can find SHAs on the [Microsoft Artifact Registry](https://mcr.microsoft.com) site. For example, at https://mcr.microsoft.com/en-us/product/dotnet/framework/runtime/tags, you can see the SHA by expanding the tag you want to see:

![image](https://user-images.githubusercontent.com/15789599/201191125-40c16ed5-75a7-476f-bd9a-80585d9e0ddb.png)

**Reply by garfbradazEQ** [reply]
Thank you :) The MCR site was down foe an age earlier though!
